### ISSUE TITLE
[BrickSlider] move "previous-item" button for better A11y  

### ISSUE THREAD

**Issue by TitanNano** [Prograp/Custom-Elements#11]
The slot for a "previeous-item" button should be moved to the beginning of the ShadowDOM to improve accessibility.
